.github/workflows/label-automation.yml

```
@@ -0,0 +1,254 @@
+          script: |
+            github.rest.issues.addLabels({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              labels: ['pr:needs-review']
+            })
+
+      # Label draft PRs as WIP
+      - name: Label draft PRs as WIP
+        if: github.event_name == 'pull_request' && github.event.pull_request.draft == true
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)